cricket_calculation.py

```diff
@@ -1,59 +1,12 @@
-def calculate_strike_rate(total_balls: int, total_runs: int) -> float:
-    """
-    Calculate the strike rate of a batsman in cricket.
+def calculate_strike_rate(runs, balls):
+    if balls == 0:
+        return 0
+    return (runs / balls) * 100
 
-    Args:
-        batsman (dict): Dictionary containing information about the batsman.
-                        "runs" (int) and "balls" (int).
 
-    Returns:
-        float: Strike rate rounded to 2 decimal places.
-    """
-    if total_balls > 0:
-        strike_rate = (total_runs / total_balls) * 100
-        return round(strike_rate, 2)
-    else:
-        return 0.00
+def calculate_current_run_rate(balls, total_runs, total_overs):
+    return total_runs / total_overs if total_overs > 0 else 0
 
 
-def calculate_current_run_rate(
```

Comment:
```
    """
    Calculate the current run rate in a cricket match.
    Args:
        balls_faced (int): Number of balls faced by the batting team.
        total_runs (int): Total runs scored by the batting team.
        total_overs (int): Total overs bowled in the match.
    Returns:
        float: Current run rate rounded to 2 decimal places.
    """
```
Getting rid of the docstrings has made this worse. Also you don't need the `balls` argument in this function